apiv2/db_import/tests/populate_db.py

```diff
@@ -438,6 +464,24 @@ def populate_stale_tiltseries(session: sa.orm.Session) -> None:
     )
 
 
+@write_data
+def populate_stale_per_section_parameters(session: sa.orm.Session) -> PerSectionParameters:
+    session.add(
+        PerSectionParameters(
+            run_id=STALE_RUN_ID,
+            astigmatic_angle=0.1,
+            frame_id=FRAME_ID,
+            major_defocus=0.5,
+            minor_defocus=0.5,
+            phase_shift=0.5,
+            max_resolution=0.5,
+            raw_angle=0.5,
+            tiltseries_id=TILTSERIES_ID,
+            z_index=0,
+        ),
```

Comment:
With this addition, we can also test the invalid sibling of a valid run and tiltseries. 
```suggestion
      ),
      PerSectionParameters(
        run_id=RUN1_ID,
        astigmatic_angle=0.1,
        frame_id=1234,
        major_defocus=0.5,
        minor_defocus=0.5,
        phase_shift=0.5,
        max_resolution=0.5,
        raw_angle=0.5,
        tiltseries_id=TILTSERIES_ID,
        z_index=10,
      ),
```